docs/topics/mta-assessment-changes-700.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+:_content-type: REFERENCE
+[id="mta-assessment-changes-700_{context}"]
+= Assessment module introduction
+
+The *Assessment* module in {ProductFullName} offers the following functionality for assessing and analyzing applications:
+
+.Separate assessment hub
+
+A separate assessment hub includes the assessment module. The *Assessment* hub integrates tightly with the *Application inventory*.
+
+.Enhanced questionnaire capabilities
+
+The ability to use custom questionnaires has been added, questionnaires can be imported and exported. Whereas {ProductShortName} 6.2 supports only the built-in legacy questionnaire. You can design and version custom questionnaires and questions using YAML syntax, with a downloadable template provided. The syntax provides:
+
```

Comment:
Rewrite as: "Previously, {ProductShortName} supported a single, built-in questionnaire.  {ProductShortName}  7.0 not only includes that questionnaire, but also supports importing and exporting questionnaires. You can design  custom questionnaires using YAML syntax, with a downloadable template provided. The syntax provides:"